farmdata2_modules/fd2_tabs/resources/CustomTableComponent.js

```diff
@@ -89,7 +89,7 @@ let CustomTableComponent = {
                 <thead>
                     <tr class="sticky-header table-text" data-cy="table-headers">
                         <th
-                        v-if="canDelete || customButtons.length > 0"  
+                        v-if="canDelete || customButtons.length > 0 || csvName != ''"  

```

Comment:
Would it make sense to use a computed property for this so that we can ensure that it is consistent across all elements?

Reply:
I believe we have all the cases covered now and a computed property would really just encapsulate these cases to make it readable for future developer. It might be worthwhile implementing it then if it makes it easier to abstract these cases and make them more manageable and extendable in the long run instead of having to look for each of these locations whenever a new case is created.

However, that implementation goes beyond the scope of this issue. A new issue would have to be created for that proposed change.

Reply:
Sounds reasonable.  Would you create that issue?